### ISSUE TITLE
New Component: Sidebar/LeftNav

### ISSUE THREAD

**Issue by micahgodbolt** [OfficeDev/office-ui-fabric-react#6221]
### Code mockup/example

__Code Sample__

```

```

__Types__

| Name      | Type     | Default Value | Description |
|-----------|----------|---------------|-------------|
| className | string   |               | Optional class name to be added to the root |

## Steps

- [ ] Themable (Component.base/Component.styles)
- [ ] Unit tests
- [ ] Visual tests
- [ ] Documentation and examples
- [ ] Keyboard Accessible
- [ ] High Constrast Support
- [ ] RTL Support
- [ ] Design Review
- [ ] API review
- [ ] Public Preview
- [ ] Ready for Publishing

**Comment by JasonGore** [comment]
Will this supercede #5815?

**Comment by OneCyrus** [comment]
imho VSTS did a pretty great job with the leftside nav in the redesign

![image](https://user-images.githubusercontent.com/7747800/45115470-320ccd80-b150-11e8-9c14-b4ab48454de5.png)

would be nice to have this pattern supported as well.

**Comment by micahgodbolt** [comment]
@OneCyrus yes, the goal of this component will be 1) standardize the way we do sidebars using the new fluent design language and 2) make it as customizable as possible to get the same functionality with different themes and component combinations

**Comment by naicky-whiterain** [comment]
Hi there, any news?

**Comment by micahgodbolt** [comment]
designs are done, but development is currently on hold.

**Comment by ChristopherHaws** [comment]
Azure DevOps also has a different menu for their Settings pages:
![image](https://user-images.githubusercontent.com/5934505/67918782-fe161800-fb5a-11e9-995e-dc9de5e18cde.png)

**Comment by farzer** [comment]
marked

**Comment by vikramdadwal** [comment]
Is there any timelines for this feature ?

**Comment by micahgodbolt** [comment]
It is not currently planned.

**Comment by mohsinonxrm** [comment]
@micahgodbolt , I'm trying to open a link from the subMenuItems. The following works fine for the main item.
`{
          key: "collapsible-binglink-item2",
          name: "Bing",
          iconProps: { iconName: "BingLogo" },          
          onRender: (item) => {
            return (
              <SidebarButton
                key={item.key}
                text={item.name}
                iconProps={item.iconProps}
                role="menuitem"
                theme={getTheme()}
                href="https://bing.com"
                target="_blank"
              />
            );
          },
        },`

However, if I use the similar approach for a sub-item, it doesn't open the link. I've also tried just providing the href and target on the sub-item itself but it doesn't work as a link.

I've seen similar functionality implemented and work fine in Microsoft's various Admin centers Sidebars. For Example:
![image](https://user-images.githubusercontent.com/21046804/103370872-d5e9b800-4a82-11eb-9e28-8b74a3074193.png)
Clicking Power BI will open the link in a new tab.

**Comment by kotpal** [comment]
Are there any examples of FluentUI based rich dashboard examples? Much like what many admin templates on the web show

**Comment by khmakoto** [comment]
Closing since if/when we do approach a new `Nav` component the approach will probably be very different to what is discussed here.
